File: Swift/gradecalc.swift
import Foundation

let BD1 = 7
let CA  = 10
let EN3 = 9
let P3  = 8
let PI  = 9
let SO1 = 8
let SDA = 9
let TGC = 10

let credit3 = 5*TGC + 5*SO1 + 5*BD1 + 5*P3 + 4*SDA + 2*PI + 2*EN3 + 2*CA
let 🐈 = Double(credit3) / 30.0

print("\(credit3)/300:", String(format: "%.3f", 🐈))

File: R/gradecalc.r
bd1 <- 7
ca <- 10
en3 <- 9
p3 <- 8
pi <- 9
so1 <- 8
sda <- 9
tgc <- 10

credit3 <- 5 * tgc + 5 * so1 + 5 * bd1 + 5 * p3 + 4 * sda + 2 * pi + 2 * en3 + 2 * ca
medie3 <- credit3 / 30

print(paste(credit3, "/300: ", format(round(medie3, 3), nsmall = 3), sep = ""))